new-log-viewer/src/components/Layout.tsx

```diff
@@ -133,31 +150,211 @@ const ConfigForm = () => {
                                 {field.label}
                             </td>
                             <td>
-                                <input
-                                    defaultValue={field.initialValue}
-                                    name={field.name}
-                                    size={100}
-                                    type={field.type}/>
+                                <Input
+                                    slotProps={{
+                                        input: {
+                                            defaultValue: field.initialValue,
+                                            name: field.name,
+                                            size: 100,
+                                            type: field.type,
+                                        },
+                                    }}/>
                             </td>
                         </tr>
                     ))}
                 </tbody>
             </table>
             <div>
-                <button type={"submit"}>Apply</button>
-                <button type={"reset"}>Clear localStorage</button>
+                <Button
+                    color={"success"}
```

Comment:
It seems the button is in blue colour, which looks the same as the other buttons. Why are we setting the color as "success" here? If we want the success button to look different than the other ones, shall we be more specific in the theme customization?